StickyTabBarViewController/Classes/StickyViewControllerSupportingTabBarController.swift

```diff
@@ -38,7 +38,7 @@ open class StickyViewControllerSupportingTabBarController: UITabBarController, S
         guard collapsableVCFlow == nil else {
             return
         }
-        childViewController.loadView()
+        childViewController.loadViewIfNeeded()
```

Comment:
I remember we discussed this & said we will keep it "for now", no? 😆 
I was not aware of `loadViewIfNeeded` but that sounds more reasonable 👍

Reply:
Haha yes, I also didn't know about this back then 😅 @Thurman1776-